src/EditorFeatures/Core/PublicAPI.Unshipped.txt

```diff
@@ -1 +1,2 @@
-
+Microsoft.CodeAnalysis.Editor.Peek.IPeekableItemFactory
```

Comment:
Hmm, is this a breaking change since we're moving it from an assembly that doesn't exist anymore? I think I'm OK with that but we should be aware of the potential for having to put a type forward back in.

Reply:
(or consider making this internal if we're going to break things anyways)